src/modules/endpoint/MemoryDatabase.js

```diff
@@ -29,6 +29,29 @@ type ContentCategoryJsonType = {|
   hash: string
 |}
 
+type ContentEventJsonType = {|
+  path: string,
+  title: string,
+  content: string,
+  last_update: string,
+  thumbnail: string,
+  available_languages: { [code: string]: string },
+  hash: string,
+  excerpt: string,
+  date: {
+    start_date: string,
+    end_date: string,
+    all_day: boolean
+  },
+  location: {
+    address: string,
+    town: string,
+    postcode: ?string,
+    latitude: ?string,
+    longitude: ?string
+  }
```

Comment:
should we make this an exact object type here (also above)?

Reply:
Hm I kind of want that this is the default behaviour, but yes yes :D